enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/SpringBootHealthCheckEnricher.java

```diff
@@ -42,6 +44,12 @@
     private static final String SCHEME_HTTPS = "HTTPS";
     private static final String SCHEME_HTTP = "HTTP";
 
+    private enum Config implements Configs.Key {
+        timeoutSeconds {{ d = "1"; }};
```

Comment:
Could we omit a default value and, if timeoutSeconds is not configured, simply don't add it to the ProbeBuilder ? That way we ensure that we will always take the default of Kubernetes and do not sync the 1s here.